src/objects/entities/Ben3.ts

```diff
@@ -184,6 +217,12 @@ class Ben3 extends Entity {
     }
   }
 
+  turnDirection(rateOfChange: number) {
+    this.gameObject.setAngularVelocity(rateOfChange);
+    // Set angular veloctiy on head so it moves with the body when it gets bigger. If we scale rateOfChange with the head it will just spin uncontrollably.
+    this.head.setAngularVelocity(rateOfChange);
```

Comment:
can we rename `rateOfChange` to `angularVelocity`?

Reply:
done